homeassistant/components/alarm_control_panel/alarmdecoder.py

```diff
@@ -7,21 +7,41 @@
 import asyncio
 import logging
 
+import voluptuous as vol
+
+from homeassistant.core import callback
 import homeassistant.components.alarm_control_panel as alarm
+import homeassistant.helpers.config_validation as cv
 from homeassistant.components.alarmdecoder import (
     DATA_AD, SIGNAL_PANEL_MESSAGE)
 from homeassistant.const import (
-    STATE_ALARM_ARMED_AWAY, STATE_ALARM_ARMED_HOME, STATE_ALARM_DISARMED,
-    STATE_ALARM_TRIGGERED)
+    ATTR_CODE, STATE_ALARM_ARMED_AWAY, STATE_ALARM_ARMED_HOME,
+    STATE_ALARM_DISARMED, STATE_ALARM_TRIGGERED)
 
 _LOGGER = logging.getLogger(__name__)
 
 DEPENDENCIES = ['alarmdecoder']
 
+SERVICE_ALARM_TOGGLE_CHIME = 'alarm_toggle_chime'
```

Comment:
Prefix the string with the platform name, ie `'alarmdecoder_alarm_toggle_chime' `.